packages/mdc-theme/README.md

```diff
@@ -111,7 +111,7 @@ CSS Custom property | Description
 
 ### CSS Classes
```

Comment:
I would just replace this entire CSS Classes section with a section about our Sass color mixins

Reply:
I am a bit worried doing so since we have only parameterized 2 elements: `button` and `fab`. The majority of the library is still not using the sass mixin way for customize theming. Could we consider mention this possibility and redirect our users to the documentation for each component instead?

Here is what I am thinking
> ### CSS Classes
>
>MDC-Web provides SASS mixins to some components to help users do customization, such as `@mixin mdc-button-filled-accessible($container-fill-color)`. For more documentation on these modifier classes, consult the documentation for each component.
>
>If you want to modify an element, which is not a Material Design component, you can apply the following modifier CSS classes.
> ....

How do you feel about this approach?

Reply:
(Continuing this review for Lynn who's feeling under the weather. Hope you feel better soon, Lynn!)

I think this sounds reasonable for the time being. I agree RE replacing the current first paragraph since we'll be tending away from offering `--primary`/etc. classes, and I also agree it's a bit early to completely remove/retool this section since we've only tackled two components thus far.

Suggestion on wording for the new paragraph: "MDC-Web provides SASS mixins, such as `mdc-button-filled-accessible`, to facilitate customization of some components. For more information, consult the documentation for each component."

Reply:
SG! Thanks Ken for helping out the review